content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -1,75 +1,44 @@
 ---
 title: 'Deploy to Cloudflare Workers'
 metaTitle: 'Deploy to Cloudflare Workers'
-metaDescription: 'Learn how to deploy a TypeScript application to Cloudflare Workers that connects to MongoDB Atlas.'
+metaDescription: 'Learn how to deploy a TypeScript application to Cloudflare Workers that connects to PostgreSQL.'
 ---
 
 <TopBlock>
 
-Today you'll be building and deploying a Cloudflare Worker that uses Prisma to save every request to a MongoDB database for inspection later.
+Today you'll be deploying a Cloudflare Worker that uses Prisma to save every request to a PostgreSQL database for inspection later.
 
-This guide covers Prisma, TypeScript, MongoDB, Data Proxy, and Cloudflare Workers from the ground up. Let's get started!
+This guide covers Prisma, TypeScript, PostgreSQL, Prisma Accelerate, and Cloudflare Workers. Let's get started!
 
 <Admonition type="warning">
 
 **Note**<br /><br />
-We are continuously testing the Data Proxy with Cloudflare Workers. [Open an issue](https://github.com/prisma/prisma/issues/new/choose) if you run into problems or have questions about deploying to Cloudflare Workers.
+We are continuously testing Prisma Accelerate with Cloudflare Workers. [Open an issue](https://github.com/prisma/prisma/issues/new/choose) if you run into problems or have questions about deploying to Cloudflare Workers.
 
 </Admonition>
```

Comment:
What does this mean?

Reply:
@janpio, thanks; this section should not exist; skipped my eyes.